app/lib/features/chat/widgets/room_avatar.dart

```diff
@@ -32,26 +33,54 @@ class RoomAvatar extends ConsumerWidget {
   Widget chatAvatarUI(Convo convo, WidgetRef ref) {
     //Data Providers
     final convoProfile = ref.watch(chatProfileDataProvider(convo));
+    final canonicalParent =
+        ref.watch(canonicalParentProvider(convo.getRoomIdStr()));
```

Comment:
I feel like that is probably something we want the caller to be able to configure. As at least in the space-chats we do not want to show parent-badges .... and if I am understanding this correctly, we would do this now.